styles/blue2_theme.css

```diff
@@ -134,3 +136,14 @@ a:hover {
   /* thanks https://codepen.io/sosuke/pen/Pjoqqp for generating this filter (the first two properties set the icon to black first to achieve a truer color match) */
   filter: brightness(0) saturate(100%) invert(25%) sepia(11%) saturate(6019%) hue-rotate(315deg) brightness(97%) contrast(89%);
 }
+
+
+
+/***** Page-specific: program.php *****/
+div.mutualEvent {
```

Comment:
thought I'd get a better match for the program backgrounds to this theme, rather than continue using the default colors that only sort of match